cmd/static-server/main.go

```diff
@@ -80,17 +80,20 @@ func loadStaticNMState(env *env.EnvInputs, nmstateDir string, imageServer imageh
 			return errors.WithMessagef(err, "problem generating ignition %s", f.Name())
 		}
 
-		imageName, ok := imageMapping[f.Name()]
-		if !ok {
-			imageName = strings.Replace(f.Name(), ".yaml", ".iso", 1) // master-1.yaml -> master-1.iso
-			log.Info("image mapping not available, using image", "name", imageName)
+		for _, suffix := range []string{".iso", ".initramfs"} {
```

Comment:
is this in the for loop? it might be github diff confusing me, but it looks like it outside the file for loop and you are using f.Name() still.. please check..

Reply:
It's the GitHub diff, they can't even manage to get the indentation to match between the changes and the context.

Here is the whole file: https://github.com/openshift/image-customization-controller/blob/628ac75c05aeee4d5a1d069017cfa30cd4b386cc/cmd/static-server/main.go#L83-L96